src/Dom/Browser.Dom.fs

```diff
@@ -808,6 +808,32 @@ type [<AllowNullLiteral>] TreeWalkerType =
 type [<AllowNullLiteral>] BarProp =
     abstract visible: bool with get, set
 
+[<StringEnum; RequireQualifiedAccess>]
```

Comment:
If you use `StringEnum(CaseRules.KebabCase)` you don't need to use `CompiledName` in every case :)

Reply:
Ups, actually I just realized that Browser.Dom was using an old Fable.Core version that didn't allow this, I've bumped the version 👍